guides/v2.1/frontend-dev-guide/cache_for_frontdevs.md

```diff
@@ -56,4 +56,4 @@ You can clean generated static view files in any of the following ways:
 
 ## Clean static files {#clean_static}
 
-Besides the cached files, in theme development process developers also deal with other saved files - static view files that are preprocessed and published to the `var/view_preprocessed` and `pub/static` directories correspondingly. In most cases when working on a custom theme, for example, if you are only working on styles, you do not need to clean cache, but need to clean the previously preprocessed and published static view files. To clean them, run  `grunt clean <theme>` or manually clear the `pub/static` and `var/view_preprocessed` directories.
+Besides the cached files, in theme development process developers also deal with other saved files - static view files that are preprocessed and published to the `var/view_preprocessed` and `pub/static` directories correspondingly. In most cases when working on a custom theme, for example, if you are only working on styles, you do not need to clean cache, but need to clean the previously preprocessed and published static view files. To clean them, run  `grunt clean <theme>` or manually clear the `pub/static` and `var/view_preprocessed` directories. After run `exec` command `clear` Magento `cache`, then run `watch` command. (It will help When we run exec some of the added custom jQuery not loading like product sliders, banners etc)
```

Comment:
@abrarpathan19, I know that you only added the last sentence, but I recommend a complete rewrite of the paragraph. 

Also, in the sentence that you added, the `exec` command has not been properly introduced in the context of cleaning static files. It seems to appear randomly without explanation. Are you sure that this is the right place to add info about this command? What about https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/css_debug.html#grunt_commands instead?

**Proposed rewrite:**

> In addition to cached files, theme developers also deal with other saved files, such as static view files that are preprocessed and written to the `var/view_preprocessed` and `pub/static` directories. 

> If you are only working on styles, you do not need to clean the cache (in most cases). However, you must clean previously preprocessed and published static view files. Run  `grunt clean <theme>` or manually clear the `pub/static` and `var/view_preprocessed` directories. 

> If you have custom jQuery that fails to load after running the `grunt exec` command, such as product sliders and banners, clean the Magento cache and run the `grunt watch` command.

Reply:
Hello @jeff-matthews ,
I have a rewrite of the paragraph. and trying to more specify the `exec` command. also, I recommend this content over here because this paragraph is related to the theme development process and Clean static files. 

Thanks for the suggestions.